docs/configuration/customizing_licensee.md

```diff
@@ -0,0 +1,13 @@
+# Customize Licensee's behavior
+
+Licensed uses [Licensee](https://github.com/licensee/licensee) to detect and evaluate OSS licenses for project dependencies found during source enumeration.  Licensed can optionally [customize Licensee's behavior](https://github.com/licensee/licensee/blob/jonabc-patch-1/docs/customizing.md#customizing-licensees-behavior) based on options set in the configuration file.
```

Comment:
I just noticed you link to your own patch branch, probably should be linked to the main repo?

Reply:
ah yeah good call